test/module/shared_model/bindings/BuilderTest.java

```diff
@@ -0,0 +1,145 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigInteger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class BuilderTest {
+    static {
+        try {
+            System.loadLibrary("iroha");
+        } catch (UnsatisfiedLinkError e) {
+            System.err.println("Native code library failed to load. \n" + e);
+            System.exit(1);
+        }
+    }
+
+    private Keypair keys;
+    private ModelBuilder builder;
+
+    ModelBuilder base() {
+        return new ModelBuilder().txCounter(BigInteger.valueOf(123))
+                .createdTime(BigInteger.valueOf(System.currentTimeMillis()))
+                .creatorAccountId("admin@test");
+    }
+
+    void setAddPeer() {
+        builder.addPeer("123.123.123.123", keys.publicKey());
+    }
+
+    @BeforeEach
+    void setUp() {
+        keys = new ModelCrypto().generateKeypair();
+        builder = base();
+    }
+
+    @Test
+    void emptyTx() {
+        ModelBuilder builder = new ModelBuilder();
+        assertThrows(IllegalArgumentException.class, builder::build);
+    }
+
+    @Test
+    void txWithoutCommand() {
+//        broken for now
```

Comment:
What does it mean? What there is need to do for fix it?

Reply:
For some reason current shared_model doesn't allow to build tx w/o any command. That's a point for discussion primarily. If it's fine I can just delete this test

Reply:
Some time ago we decided that tx without commands is invalid (in chat discussion I believe) and I did a check in validator for this case.

Reply:
Ok, I'll delete it then